x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/client_libraries_popover/popover.test.tsx

```diff
@@ -85,11 +85,11 @@ describe('ClientLibrariesPopover', () => {
       wrapper
         .find(EuiContextMenuPanel)
         .prop('items')
-        ?.map((item) => shallow(<div>{item}</div>)) || [];
+        ?.map((item: HTMLElement) => shallow(<div>{item}</div>)) || [];
 
     expect(contextMenuItems.length > 0).toBeTruthy();
 
-    contextMenuItems.forEach((item, index) => {
+    contextMenuItems.forEach((item: any, index: number) => {
```

Comment:
Kibana actively tracks `any` usages as a negative mark against teams. We should avoid introducing them where possible.

```suggestion
    contextMenuItems.forEach((item: ShallowWrapper, index: number) => {
```

`ShallowWrapper` is imported from `enzyme`